pages/listings/listing/edit/index.html

```diff
@@ -33,15 +33,15 @@ <h1 class="text-center mt-4">Edit Listing</h1>
               </div>
               <div class="form mb-3">
                 <label for="location">Location</label>
-                <input class="form-control bg-theme-light" style="border-radius: 0px" type="text" name="location" id="createLocation" required />
+                <input class="form-control bg-theme-light" style="border-radius: 0px" type="text" name="location" id="createLocation" required  aria-label="Enter location" />
               </div>
               <div class="form mb-3">
                 <label for="deadline">Application Deadline</label>
-                <input class="form-control bg-theme-light" style="border-radius: 0px" type="datetime-local" name="deadline" id="createDeadline" required title="Enter a valid date" />
+                <input class="form-control bg-theme-light" style="border-radius: 0px" type="datetime-local" name="deadline" id="createDeadline" required title="Enter a valid date"  aria-label="Application deadline" />
               </div>
               <div class="input-group mt-5 mb-3">
                 <span class="input-group-text" style="border-radius: 0px">Upload Cover Picture</span>
-                <input type="text" class="form-control bg-theme-light" style="border-radius: 0px" id="pictureUrl" />
+                <input type="text" class="form-control bg-theme-light" style="border-radius: 0px" id="pictureUrl"  aria-label="Upload cover picture" />
```

Comment:
From my understanding we are using URL's for cover pictures rather than uploading images
Therefore this aria-label should also be changed to "Enter URL for cover picture"

Reply:
changed it now.